bodo/tests/test_df_lib/test_end_to_end.py

```diff
@@ -397,6 +397,43 @@ def test_filter_string(datapath):
     )
 
 
+@pytest.mark.parametrize(
+    "op", [operator.eq, operator.ne, operator.gt, operator.lt, operator.ge, operator.le]
+)
+def test_filter_datetime(datapath, op):
```

Comment:
Is the pushdown case a follow up? Filtering date/time is very common in practice.

Reply:
Pushdown already working.  Test added.